app/views/items/show.html.haml

```diff
@@ -12,7 +12,16 @@
         = link_to @item.url, target: :_blank do
           = item_thumbnail(@item, request)
         = link_to @item.url, target: :_blank, class:'ViewObject' do
-          View Object
+          Get full
+          - if @item.type.present? and @item.type.is_a? String
+            = @item.type
+          - else
+            object
```

Comment:
`item` works for me.  There has been no discussion so far about `item` vs. `object`, but I like `item`.  I'll make the change.